packages/os/healthdog.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Send a Healthdog Ping
+
+[Service]
+Type=oneshot
+RemainAfterExit=false
+ExecStart=/bin/healthdog send-health-ping
```

Comment:
```suggestion
ExecStart=/usr/bin/healthdog send-health-ping
```

Reply:
Does this mean I should change the install directives as well in os.spec?

Reply:
In spec files you'd be using macros, so "/bin" vs. "/usr/bin" shouldn't appear anyway.

Reply:
I didn't realize the program could be called using either path, but it seems to work with no changes to os.spec.

Addressed in [f18d841](https://github.com/bottlerocket-os/bottlerocket/compare/ec0abbf943fe3806294f53fd4e118be3d1f529fc..f18d841ee1c2b35d5f09136ea3f7136e2a4d65ae)